### PR TITLE
Update CHANGELOG for 0.8.0

### PR DESCRIPTION
**Pull request by angelozerr**
Update CHANGELOG for 0.8.0

Signed-off-by: azerr <azerr@redhat.com>

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,68 @@
 # Change Log
 
+## [0.8.0](https://github.com/redhat-developer/vscode-xml/milestone/9?closed=1) (July 23, 2019)
+
+### Enhancements
+
+ * Find definition for start/end tag element. See[#535](https://github.com/angelozerr/lsp4xml/issues/535)
+ * Improve Source information from completion. See[#529](https://github.com/angelozerr/lsp4xml/issues/529)
+ * Completion should support markdown for documentation. See[#526](https://github.com/angelozerr/lsp4xml/issues/526)
```
Comment (xorye) [inline CHANGELOG.md]: Markdown support for completion documentation

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,68 @@
...
+ * Support selective enablement of outline per file. See[#427](https://github.com/angelozerr/lsp4xml/issues/427)
+ * `.ent` and `.mod` files should be parsed as DTDs. See[#380](https://github.com/angelozerr/lsp4xml/issues/380)
+ * Add support for `textDocument/typeDefinition` from XML to XMLSchema/DTD . See[#371](https://github.com/angelozerr/lsp4xml/issues/371)
+ * Show possible elements for completion based on XML Schema. See[#347](https://github.com/angelozerr/lsp4xml/issues/347)
+ * Add support for `textDocument/definition` for XML Schema. See[#148](https://github.com/angelozerr/lsp4xml/issues/148)
+ * Add support for `textDocument/references` for XML Schema types. See[#58](https://github.com/angelozerr/lsp4xml/issues/58)
+ * Add support for `textDocument/codelens` for XML Schema types. See[#55](https://github.com/angelozerr/lsp4xml/issues/55)
+ * Add completion for `comment` and `#region`. See[#54](https://github.com/angelozerr/lsp4xml/issues/54)
+ * CData block auto-complete. See[#168](https://github.com/redhat-developer/vscode-xml/issues/168)
+ * Active editor cannot provide outline information. See[#166](https://github.com/redhat-developer/vscode-xml/issues/166)
```
Comment (xorye) [inline CHANGELOG.md]: Add XML support for unsaved XML, XSD and XSL files. See [#156]

Seems like fixing #166 and #154 is directly related to that ^ enhancement.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,68 @@
...
+ * Error range for `src-ct.1`. See[#453](https://github.com/angelozerr/lsp4xml/issues/453)
+ * Error range for `duplicate attribute`.. See[#452](https://github.com/angelozerr/lsp4xml/issues/452)
+ * Error range for `p-props-correct.2.1`. See[#436](https://github.com/angelozerr/lsp4xml/issues/436)
+ * Element/Attribute names must be case sensitive. See[#433](https://github.com/angelozerr/lsp4xml/issues/433)
+ * Selection formatting ignores attribute indentation preference. See[#429](https://github.com/angelozerr/lsp4xml/issues/429)
+ * Error range for `cos-all-limited.2`. See[#428](https://github.com/angelozerr/lsp4xml/issues/428)
+ * Error range for `src-element.3`. See[#420](https://github.com/angelozerr/lsp4xml/issues/420)
+ * Documents with an Internal Subset DOCTYPE shouldn't stop trying to bind. See[#379](https://github.com/angelozerr/lsp4xml/issues/379)
+ * When DTD is not found, XML is not checked for well-formedness. See[#167](https://github.com/redhat-developer/vscode-xml/issues/167)
+ * unexpected indentation after empty element + attribute. See[#159](https://github.com/redhat-developer/vscode-xml/issues/159)
```
Comment (xorye) [inline CHANGELOG.md]: Unexpected indentation after empty element and attribute

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,68 @@
...
+ * Formatter inserts spaces in empty lines. See[#157](https://github.com/redhat-developer/vscode-xml/issues/157)
+ * Plugin is not recognized as a XML formatter. See[#154](https://github.com/redhat-developer/vscode-xml/issues/154)
+ * vscode not reloading and revalidating XML/XSD files modified outside itself. See[#131](https://github.com/redhat-developer/vscode-xml/issues/131)
```
Comment (xorye) [inline CHANGELOG.md]: VSCode was not revalidating XML files when relevant XSD files were modified outside VSCode.

File: CHANGELOG.md (hunk 1 above)
Comment (angelozerr) [reply CHANGELOG.md]: I remobed this issue since https://github.com/redhat-developer/vscode-xml/issues/32 covers it

File: CHANGELOG.md (hunk 2 above)
Comment (angelozerr) [reply CHANGELOG.md]: I add this 156 issue, I forget it because it was not flagged as 0.8.0

**Comment by angelozerr** [comment]
@xorye done

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,68 @@
 # Change Log
 
+## [0.8.0](https://github.com/redhat-developer/vscode-xml/milestone/9?closed=1) (July 23, 2019)
+
+### Enhancements
+
+ * `Markdown` support for `completion` and `hover` documentation. See [#32](https://github.com/redhat-developer/vscode-xml/issues/32)
+ * Add completion for `comment` and `#region`. See [#54](https://github.com/angelozerr/lsp4xml/issues/54)
```
Comment (fbricon) [inline CHANGELOG.md]: all links to lsp4xml issues should be prefixed with lsp4xml, eg. 
```
[lsp4xml#54](https://github.com/angelozerr/lsp4xml/issues/54)
```

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,5 +1,68 @@
...
+ * Add XML support for unsaved XML, XSD and XSL files. See [#156](https://github.com/redhat-developer/vscode-xml/issues/156) 
+ * Trigger `textDocument/didSave` if an XSD file was changed externally from a different editor . See [#132](https://github.com/redhat-developer/vscode-xml/issues/132)
+ * Improved XML validation when XSD files are saved. See [#506](https://github.com/angelozerr/lsp4xml/issues/506)
+
+### Bug Fixes
+
+ * Hover markup response ignored the hover client capability. See [#525](https://github.com/angelozerr/lsp4xml/issues/525)
+ * Completion capability was lost in specific scenarios. See [#522](https://github.com/angelozerr/lsp4xml/issues/522)
+ * Fixed NPE in  `textDocument/definition` in XSD files. See [#488](https://github.com/angelozerr/lsp4xml/issues/488)
+ * Fix test with markdown on Windows OS.. See [#487](https://github.com/angelozerr/lsp4xml/issues/487)
```
Comment (fbricon) [inline CHANGELOG.md]: extra .

Comment (angelozerr) [reply CHANGELOG.md]: @fbricon updated

File: CHANGELOG.md (hunk 5 above)
Comment (angelozerr) [reply CHANGELOG.md]: @fbricon updated

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,5 +1,68 @@
...
+ * Parse `.ent` and `.mod` files as DTD files. See [lsp4xm#380](https://github.com/angelozerr/lsp4xml/issues/380)
+ * Add support for `textDocument/typeDefinition` from XML to XMLSchema/DTD. See [lsp4xm#371](https://github.com/angelozerr/lsp4xml/issues/371)
+ * Add support for `textDocument/definition` for XML Schema. See [lsp4xm#148](https://github.com/angelozerr/lsp4xml/issues/148)
+ * Add support for `textDocument/references` for XML Schema types. See [lsp4xm#58](https://github.com/angelozerr/lsp4xml/issues/58)
+ * Add support for `textDocument/codelens` for XML Schema types. See [lsp4xm#55](https://github.com/angelozerr/lsp4xml/issues/55)
+ * Add support for clickable`XSD CodeLens`. See [lsp4xm#490](https://github.com/angelozerr/lsp4xml/issues/490)
+ * Active editor cannot provide outline information. See [#166](https://github.com/redhat-developer/vscode-xml/issues/166)
+ * Add XML support for unsaved XML, XSD and XSL files. See [#156](https://github.com/redhat-developer/vscode-xml/issues/156) 
+ * Trigger `textDocument/didSave` if an XSD file was changed externally from a different editor . See [#132](https://github.com/redhat-developer/vscode-xml/issues/132)
```
Comment (fbricon) [inline CHANGELOG.md]: either this one or bug 131 should be removed, as one is a duplicate of the other

File: CHANGELOG.md [hunk 8]
```diff
@@ -1,5 +1,68 @@
...
+ * Fixed error range for `p-props-correct.2.1`. See [lsp4xm#436](https://github.com/angelozerr/lsp4xml/issues/436)
+ * Fixed error range for `cos-all-limited.2`. See [lsp4xm#428](https://github.com/angelozerr/lsp4xml/issues/428)
+ * Fixed error range for `src-element.3`. See [lsp4xm#420](https://github.com/angelozerr/lsp4xml/issues/420)
+ * Documents with an Internal Subset DOCTYPE had stopped trying to bind. See [lsp4xm#379](https://github.com/angelozerr/lsp4xml/issues/379)
+ * XML did not validate when bounded DTD file was not found. See [#167](https://github.com/redhat-developer/vscode-xml/issues/167)
+ * Unexpected indentation after empty element and attribute. See [#159](https://github.com/redhat-developer/vscode-xml/issues/159)
+ * Formatter inserts spaces in empty lines. See [lsp4xm#157](https://github.com/redhat-developer/vscode-xml/issues/157)
+ * Plugin is not recognized as a XML formatter. See [#154](https://github.com/redhat-developer/vscode-xml/issues/154)
+ * VSCode was not revalidating XML files when relevant XSD files were modified outside VSCode. See [#131](https://github.com/redhat-developer/vscode-xml/issues/131)
```
Comment (fbricon) [inline CHANGELOG.md]: either this one or enhancement 132 should be removed, as one is a duplicate of the other

File: CHANGELOG.md (hunk 7 above)
Comment (fbricon) [reply CHANGELOG.md]: `editor.` no space between editor and .

File: CHANGELOG.md (hunk 8 above)
Comment (angelozerr) [reply CHANGELOG.md]: @fbricon updated

**Pull request merged**